Sources/NetworkProtection/Storage/NetworkProtectionTunnelErrorStore.swift

```diff
@@ -35,18 +37,31 @@ public final class NetworkProtectionTunnelErrorStore {
         self.notificationCenter = notificationCenter
     }
 
+    // MARK: - Posting Distributed Notifications
+
+    private func postLastErrorMessageChangedNotification(_ errorMessage: String?) {
+        notificationCenter.post(.tunnelErrorChanged, object: errorMessage)
+    }
+
+#else
+
+    public init(userDefaults: UserDefaults = .standard) {
+        self.userDefaults = userDefaults
+    }
+
+#endif
+
```

Comment:
This code could be much nicer IMO if we had `MacNetworkProtectionTunnelErrorStore` sending out the distributed notifications and composing internally `NetworkprotectionTunnelErrorStore`. 

But before doing that I think we may want to discuss a bit where we want the MacOS code to live.